spec/request/in_progress_forms_request_spec.rb

```diff
@@ -307,6 +306,32 @@ def response_body
           )
         end
 
+        it 'can have nil metadata' do
+          put v0_in_progress_form_url(new_form.form_id),
+              params: { form_data: { greeting: 'Hello!' } }.to_json,
+              headers: { 'CONTENT_TYPE' => 'application/json' }
+          expect(response).to have_http_status(:ok)
+        end
+
+        it 'can\'t have nil formData' do
+          put v0_in_progress_form_url(new_form.form_id)
+          expect(response).to have_http_status(:error)
+        end
+
+        it 'can\'t have non-hash formData' do
```

Comment:
Instead of escaping, could you use `"` instead please?

Reply:
No prob! For some reason i though our rubocop settings were enforcing single quotes when not interpolating